index.html

```diff
@@ -6987,7 +7015,15 @@
           </td>
         </tr>
 
-
+      <tr id="def_Mukawwar">
+        <td>Mukawwar</td>
```

Comment:
Like our other words, this should probably be in lower case. (“mukawwar” instead of “Mukawwar”.)